src/multivariate.jl

```diff
@@ -55,6 +57,12 @@ function _logpdf(d::MvNormal, x::Union{Tracker.TrackedVector, Tracker.TrackedMat
     _logpdf(TuringMvNormal(d.μ, getchol(d.Σ)), x)
 end
 
+import StatsBase: entropy
+function entropy(d::TuringDiagNormal)
```

Comment:
Can we add a test for this?

Reply:
Will do!

Reply:
Sorry, completely forgot about this! Done diddit now though:)